src/resource/ensemble.rs

```diff
@@ -15,6 +23,14 @@ resource! {
         /// The current status of this ensemble.
         pub status: GenericStatus,
 
+        /// Maps BigML field codes to named input fields
+        /// This is just a single-entry dictionary of "fields" key mapping to
+        /// dictionary of fields
+        pub ensemble: HashMap<String, HashMap<String, EnsembleField>>,
+
+        /// Maps average importance per field (BigML field id => importance value).
+        pub importance: HashMap<String, f64>,
```

Comment:
There's a pretty good chance that we'll need to make these two fields `Option<HashMap<...>>` and set `#[serde(default)]`, because they might not be present while the ensemble is still being constructed. Or at least many other parts of the API work that way.